db/migrate/20181031143515_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
might be an idea to add a unique index on `auith_id` given we intend to look up users based on it.

Reply:
yes, will do.